addons/resource.language.en_gb/resources/strings.po

```diff
@@ -22496,17 +22508,57 @@ msgstr ""
 
 #. Value of setting - MByte
 #: xbmc/settings/SevicesSettings.cpp
+#: xbmc/settings/PlayerSettings.cpp
 msgctxt "#37122"
 msgid "{0:d} MB"
 msgstr ""
 
 #. Value of setting - GByte
 #: xbmc/settings/SevicesSettings.cpp
+#: xbmc/settings/PlayerSettings.cpp
 msgctxt "#37123"
 msgid "{0:d} GB"
 msgstr ""
 
-#empty strings from id 37124 to 38010
+#empty strings from id 37124 to 37127
+
+#. Value of setting - second
+#: xbmc/settings/PlayerSettings.cpp
+msgctxt "#37128"
+msgid "{0:d} second"
+msgstr ""
+
+#. Value of setting - seconds
+#: xbmc/settings/PlayerSettings.cpp
+msgctxt "#37129"
+msgid "{0:d} seconds"
+msgstr ""
+
+#. Setting "Audio/video queue time"
+#: system/settings/settings.xml
+msgctxt "#37130"
+msgid "Audio/video queue time"
+msgstr ""
+
+#. Description of setting with label #37130 "Audio/video queue time"
+#: system/settings/settings.xml
+msgctxt "#37131"
+msgid "Duration in seconds of audio/video queues. The duration determines the amount of data stored in memory."
+msgstr ""
+
+#. Setting "Video queue maximum size"
+#: system/settings/settings.xml
+msgctxt "#37132"
+msgid "Video queue maximum size"
+msgstr ""
+
+#. Description of setting with label #37132 "Video queue maximum size"
+#: system/settings/settings.xml
+msgctxt "#37133"
+msgid "Limits the maximum memory usage for the video queue. The amount of memory used depends on the video bitrate and the length of the queue [memory (MB) = bitrate (Mbit/s) * seconds / 8]. If the memory limit is reached, the queue time is reduced as necessary."
```

Comment:
I would remove the math. I think it's a good description, and anyone who is messing with this setting knows the Mbit/s to MB conversion intuitively. The math just adds to the "wall of text".